operations/iuf/IUF.md

```diff
@@ -591,7 +591,7 @@ INFO              BEGIN: preflight-checks(0)
 
 ### Log files
 
-IUF stores detailed information in log files which are stored on a Ceph Block Device typically mounted at `/etc/cray/upgrade/`. The default log file directory location can be overridden with the `iuf -b` and `iuf --log-dir`
+IUF stores detailed information in log files which are stored on a Ceph block device typically mounted at `/etc/cray/upgrade/`. The default log file directory location can be overridden with the `iuf -b` and `iuf --log-dir`
```

Comment:
`/etc/cray/upgrade/` is the directory where the mount point resides, but it looks to me like the actual root of the mount is `/etc/cray/upgrade/csm` (at least on tyr)

Reply:
Fixed in next commit.